command/rootchain/helper/utils.go

```diff
@@ -159,9 +159,9 @@ func GetECDSAKey(privateKey, accountDir, accountConfig string) (ethgo.Key, error
 // GetValidatorInfo queries SupernetManager smart contract on root
 // and retrieves validator info for given address
 func GetValidatorInfo(validatorAddr ethgo.Address, supernetManagerAddr, stakeManagerAddr types.Address,
```

Comment:
Since `SupernetManager` and `StakeManager` are now one contract, we don't need both addresses here. Please revise this function, it doesn't seem correct to me now. Plus since `StakeManager` is deployed on genesis, we can use its hardcoded address here, so no need to pass any address to function.